Ctor 457 new pack network juniper mseries netconf (#4416)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-firewalls-juniper-ssg-snmp.md

> import Tabs from '@theme/Tabs';

Reviewer: Juniper SSG SNMP?

Reviewer 2: SAAS-1970

import TabItem from '@theme/TabItem';

## Dépendances du connecteur de supervision

Les connecteurs de supervision suivants sont automatiquement installés lors de l'installation du connecteur **Juniper SSG** 

> depuis la page **Configuration > Gestionnaire de Connecteurs de supervision** :

Reviewer: ```suggestion
depuis la page **Configuration > Connecteurs  > Connecteurs de supervision** :
```

#### Découverte d'hôtes

| Nom de la règle | Description                                                                                                                                                                                                                             |
|:----------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| SNMP Agents     | Découvre les ressources via un scan réseau SNMP. Installez le connecteur [Generic SNMP](./applications-protocol-snmp.md) pour obtenir la règle de découverte et créez un modificateur pour le modèle d'hôte **Net-Juniper-SSG-custom**. |

Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.

#### Découverte de services

| Nom de la règle          | Description |
|:-------------------------|:------------|

> | Net-Juniper-SSG-Vpn-Name |             |

Reviewer 2: ```suggestion
| Net-Juniper-SSG-Vpn-Name | Découvre les VPN et les supervise avec le modèle de service Vpn-Status |
```

### Pack

> La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md). de supervision

Reviewer 2: ```suggestion
La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md) de supervision.
```

> La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md) de supervision.

Reviewer: ```suggestion
La procédure d'installation des connecteurs de supervision diffère légèrement [suivant que votre licence est offline ou online](../getting-started/how-to-guides/connectors-licenses.md).
```

1. Si la plateforme est configurée avec une licence *online*, l'installation d'un paquet

> n'est pas requise pour voir apparaître le connecteur dans le menu **Configuration > Gestionnaire de Connecteurs de supervision**.

Reviewer: ```suggestion
n'est pas requise pour voir apparaître le connecteur dans le menu **Configuration > Connecteurs > Connecteurs de supervision**.
```

</Tabs>

2. Quel que soit le type de la licence (*online* ou *offline*), installez le connecteur **Juniper SSG**

> depuis l'interface web et le menu **Configuration > Gestionnaire de Connecteurs de supervision**.

Reviewer: ```suggestion
depuis l'interface web et le menu **Configuration > Connecteurs > Connecteurs de supervision**.
```

3. Appliquez le modèle d'hôte **Net-Juniper-SSG-custom**.

> Si vous utilisez SNMP en version 3, vous devez configurer les paramètres spécifiques associés via la macro **SNMPEXTRAOPTIONS**.

> > Plus d'informations dans la section [Troubleshooting SNMP](../getting-started/how-to-guides/troubleshooting-plugins.md#snmpv3-options-mapping).

Reviewer: ```suggestion
> Plus d'informations dans la section [Troubleshooting SNMP](../getting-started/how-to-guides/troubleshooting-plugins.md#mapping-des-options-snmpv3).
```

| Macro    | Description                                     | Valeur par défaut | Obligatoire |
|:---------|:------------------------------------------------|:------------------|:-----------:|
| CRITICAL | Critical threshold in percent (1min,5min,15min) | 90,85,80          |             |

> | WARNING  | Warning threshold in percent (1min,5min,15min)  | 80,75,70          |             |

Reviewer: the old explanation was better

> | CRITICAL | Critical threshold in percent (1min,5min,15min) | 90,85,80          |             |
> | WARNING  | Warning threshold in percent (1min,5min,15min)  | 80,75,70          |             |

Reviewer 2: ```suggestion
| CRITICAL | Critical threshold in percent for the average value during the last minute, the last 5 minutes and the last 15 minutes. | 90,85,80          |             |
| WARNING  | Warning threshold in percent for the average value during the last minute, the last 5 minutes and the last 15 minutes.  | 80,75,70          |             |
```

| EXTRAOPTIONS   | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).                                                                                                                              | --verbose         |             |

</TabItem>
<TabItem value="Vpn-Status" label="Vpn-Status">

| Macro              | Description                                                                                                                                                                                           | Valeur par défaut                                          | Obligatoire |
|:-------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------------------------------------|:-----------:|

> | FILTERNAME         | Filter VPN name (can be a regexp)                                                                                                                                                                     | .*                                                         |             |

Reviewer: the old explanation was better

> | FILTERNAME         | Filter VPN name (can be a regexp)                                                                                                                                                                     | .*                                                         |             |

Reviewer 2: ```suggestion
| FILTERNAME         | Check only VPNs whose name match the given regular expression                                                                                                                                                                     | .*                                                         |             |
```

| CRITICALUPDATETIME | Critical threshold for update time (in seconds)                                                                                                                                                       |                                                            |             |
| EXTRAOPTIONS       | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).                                                                                                    | --verbose                                                  |             |

</TabItem>
<TabItem value="Vpn-Usage" label="Vpn-Usage">

| Macro              | Description                                                                                        | Valeur par défaut | Obligatoire |
|:-------------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTERNAME         | Filter VPN name (can be a regexp)                                                                  | .*                |             |

Reviewer: same

> | FILTERNAME         | Filter VPN name (can be a regexp)                                                                  | .*                |             |

Reviewer 2: ```suggestion
| FILTERNAME         | Fheck only VPNs whose name match the given regular expression                                                                  | .*                |             |
```
